src/segment-loader.js

```diff
@@ -1224,6 +1224,11 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     const Cue = window.WebKitDataCue || window.VTTCue;
     const value = {
+      bandwidth: segmentInfo.playlist.attributes['BANDWIDTH'],
```

Comment:
fixing the `attributes` is something we want to address, see #1205 and https://github.com/videojs/m3u8-parser/issues/25 so we could just wait on that before merging these changes in. Also just adding `attributes` as a property to `value` would work as well I'd think, though it may be better to individually set values so the cue isn't holding on the memory reference of the playlist object. Because of the way our [PlaylistLoader works](https://github.com/videojs/videojs-contrib-hls/blob/master/src/playlist-loader.js#L75) live playlist refreshes create a deep copy of the playlist and updating the copy, instead of just updating the original reference. The `attributes` aren't going to change on refreshes, and its possible there are other places in our code that is holding onto old playlist references, but its probably a good idea to not do so if we know about it